app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">FHIR Bundle</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+          <div class="content">
+            <pre style="white-space:pre-wrap;">
+              <p> <%= JSON.pretty_generate(@health_card.to_hash) %> </p>
+            </pre>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Minified FHIR Bundle</p>
```

Comment:
Technically the FHIR Bundle isn't minified, but the JWS Payload is minified, so I think we'll want to change the title here. Also the displayed minified object here should include all the top-level JWS fields, not just the FHIR Bundle.

Reply:
https://smarthealth.cards/examples/example-00-c-jws-payload-minified.json

Here is a good example from the spec of what this should look like.